library/globals.inc.php

```diff
@@ -949,14 +949,14 @@
       xl('Exclude Statements from Printing'),
       'bool',                           // data type
       '0',                              // default = false
-      xl('This will use the Following settings to Exclude Patient Statements from Printing.')
+      xl('This will enable the Ability to Exclude Patient Statements from Printing.')
```

Comment:
Note I tried to make it more generic on the code that went into the codebase. Can change this if needed after the patient and insurance exceptions come in.